common/models/user.js

```diff
@@ -568,6 +642,83 @@ module.exports = function(User) {
    * @promise
    */
 
+  User.verify = function(uid, verifyOptions, options, cb) {
```

Comment:
I am wondering whether we need to add this method, considering that the remoting layer is exposing `User.prototype.verify` now? It would make sense to me to add `User.verify` if that was the method exposed via REST, but this way I think it's not worth the future maintenance effort.

Thoughts?